ps_facetedsearch.php

```diff
@@ -729,8 +729,12 @@ public function getContent()
         $this->context->smarty->assign('uri', $this->getPathUri());
 
         // Assign assets
-        if (method_exists($this->context->controller, 'addJquery')) {
-            $this->context->controller->addJS(_PS_JS_DIR_ . 'jquery/plugins/jquery.sortable.js');
+        if (file_exists(_PS_ROOT_DIR_ . _PS_JS_DIR_ . 'vendor/Sortable.min.js')) {
```

Comment:
@leemyongpakvn, maybe this check should be:
`_PS_JS_DIR_` and this is why it doesn't work on 8.1

Reply:
@kpodemski I have just tested again with PS develop on my localhost. PHP `file_exists` requires an absolute path, without `_PS_ROOT_DIR_`, it will returns false. PS `controller->addJS` just need `_PS_JS_DIR_` because it really depends on `Media::getJSPath` to process.
I think we need another QA test with a fresh develop branch, or will till PS 8.1.0 officially released to test gain.